pytest-testslide/setup.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from setuptools import setup  # type: ignore
+
+with open("README.md", encoding="utf8") as f:
+    readme = f.read()
+
+setup(
+    name="pytest-testslide",
+    version="1.0.0",
```

Comment:
I think we should sync this version and `testslide>=2.2.1`  below with `testslide/version`, what do you think?

Reply:
Shouldn't this be changed to`ts_version`?

Reply:
@deathowl I think this is missing to be updated to `ts_version`.